### PR TITLE
[13.0][MIG] business_requirement_sale_timesheet: Migration to 13.0

### PR DESCRIPTION
**Pull request by Tardo**
depends #335 

cc @tecnativa TT19720

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please rebase this one

**Comment by Tardo** [comment]
All green!

**Review by pedrobaeza — APPROVED** [review]

File: business_requirement_sale_timesheet/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+* `Tecnativa <https://www.tecnativa.com>`_:
+
+  * Pedro M. Baeza
+  * Victor M.M. Torres
```
Comment (joao-p-marques) [inline business_requirement_sale_timesheet/readme/CONTRIBUTORS.rst:4]: You could add your name to the Contributors list :smile:

**Review by joao-p-marques — APPROVED** [review]
LGTM :+1: 

Just a small comment

**Comment by pedrobaeza** [comment]
Let's not wait for having this finishes:

/ocabot merge nobump

**Pull request merged**